### PR TITLE
enh(mc): refresh monitoring pack procedure page

### PR DESCRIPTION
**Pull request by omercier**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 1]
```diff
@@ -134,98 +151,270 @@ yum install centreon-plugin-Cloud-Azure-DataFactory-Api
...
-    --critical-factory-size='' \
-    --warning-resource-count='' \
-    --critical-resource-count='' \
-    --use-new-perfdata
+/usr/lib/centreon/plugins/centreon_azure_datafactory_factories_api.pl \
+	--plugin=cloud::azure::datafactory::factories::plugin \
+	--mode=integration-runtime \
+	--custommode='api' \
+	--resource='' \
+	--resource-group='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:255]: rien comme paramètres

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 2]
```diff
@@ -134,98 +151,270 @@ yum install centreon-plugin-Cloud-Azure-DataFactory-Api
...
+| --source-encoding                          | Define the character encoding of the response sent by the monitored resource Default: 'UTF-8'.      Microsoft Azure Rest API      To connect to the Azure Rest API, you must register an application.      Follow the 'How-to guide' at     https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-cr     eate-service-principal-portal      The application needs the 'Monitoring Reader' role (See     https://docs.microsoft.com/en-us/azure/azure-monitor/platform/roles-perm     issions-security#monitoring-reader).      This custom mode is using the 'OAuth 2.0 Client Credentials Grant Flow'      For further informations, visit     https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-oauth     2-client-creds-grant-flow                                                                                                                                                                                   |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:350]: this is all garbled

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 3]
```diff
@@ -134,98 +151,270 @@ yum install centreon-plugin-Cloud-Azure-DataFactory-Api
...
+| --warning-$metric$  | Thresholds warning ($metric$ can be: 'factory-percentage-usage', 'resource-percentage-usage', 'factory-size', 'resource-count').     |
+| --critical-$metric$ | Thresholds critical ($metric$ can be: 'factory-percentage-usage', 'resource-percentage-usage', 'factory-size', 'resource-count').    |
+
+</TabItem>
+<TabItem value="Integration-Runtime" label="Integration-Runtime">
+
+| Option              | Description                                                                                                                                                                                                                                |
+|:--------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --resource          | Set resource name or id (Required).                                                                                                                                                                                                        |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:403]:
```suggestion
| --resource          | Set resource name or ID (Required).                                                                                                                                                                                                        |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 4]
```diff
@@ -134,98 +151,270 @@ yum install centreon-plugin-Cloud-Azure-DataFactory-Api
...
+Les options disponibles pour chaque modèle de services sont listées ci-dessous :
 
-Rendez-vous sur la [documentation dédiée](../getting-started/how-to-guides/troubleshooting-plugins.md#http-and-api-checks)
-des plugins basés sur HTTP/API.
+<Tabs groupId="sync">
+<TabItem value="Factory-Usage" label="Factory-Usage">
+
+| Option              | Description                                                                                                                          |
+|:--------------------|:-------------------------------------------------------------------------------------------------------------------------------------|
+| --resource          | Set resource name or id (Required).                                                                                                  |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:393]:
```suggestion
| --resource          | Set resource name or ID (Required).                                                                                                  |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 5]
```diff
@@ -134,98 +151,270 @@ yum install centreon-plugin-Cloud-Azure-DataFactory-Api
...
+| --resource          | Set resource name or id (Required).                                                                                                                                                                                                        |
+| --resource-group    | Set resource group (Required if resource's name is used).                                                                                                                                                                                  |
+| --filter-metric     | Filter metrics (Can be: 'IntegrationRuntimeAvailableMemory', 'IntegrationRuntimeAvailableNodeNumber', 'IntegrationRuntimeAverageTaskPickupDelay', 'IntegrationRuntimeCpuPercentage', 'IntegrationRuntimeQueueLength') (Can be a regexp).   |
+| --warning-$metric$  | Thresholds warning ($metric$ can be: 'available-memory', 'available-node-number', 'average-task-pickup-delay', 'cpu-percentage', 'queue-length').                                                                                          |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:406]:
```suggestion
| --warning-$metric$  | Warning threshold ($metric$ can be: 'available-memory', 'available-node-number', 'average-task-pickup-delay', 'cpu-percentage', 'queue-length').                                                                                          |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 6]
```diff
@@ -134,98 +151,270 @@ yum install centreon-plugin-Cloud-Azure-DataFactory-Api
...
+| --critical-$metric$ | Thresholds critical ($metric$ can be: 'available-memory', 'available-node-number', 'average-task-pickup-delay', 'cpu-percentage', 'queue-length').                                                                                         |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:407]:
```suggestion
| --critical-$metric$ | Critical threshold ($metric$ can be: 'available-memory', 'available-node-number', 'average-task-pickup-delay', 'cpu-percentage', 'queue-length').                                                                                         |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 7]
```diff
@@ -135,98 +152,267 @@ apt install centreon-plugin-cloud-azure-datafactory-api
...
+| --source-encoding                          | Define the character encoding of the response sent by the monitored resource Default: 'UTF-8'.      Microsoft Azure Rest API      To connect to the Azure Rest API, you must register an application.      Follow the 'How-to guide' at     https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-cr     eate-service-principal-portal      The application needs the 'Monitoring Reader' role (See     https://docs.microsoft.com/en-us/azure/azure-monitor/platform/roles-perm     issions-security#monitoring-reader).      This custom mode is using the 'OAuth 2.0 Client Credentials Grant Flow'      For further informations, visit     https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-oauth     2-client-creds-grant-flow                                                                                                                                                                                   |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:348]: all weird

File: pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 8]
```diff
@@ -135,98 +152,267 @@ apt install centreon-plugin-cloud-azure-datafactory-api
...
-    --critical-resource-percentage-usage='' \
-    --warning-factory-size='' \
-    --critical-factory-size='' \
-    --warning-resource-count='' \
-    --critical-resource-count='' \
-    --use-new-perfdata
+/usr/lib/centreon/plugins/centreon_azure_datafactory_factories_api.pl \
+	--plugin=cloud::azure::datafactory::factories::plugin \
+	--mode=integration-runtime \
+	--custommode='api' \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:251]: no parameters

File: pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 9]
```diff
@@ -135,98 +152,267 @@ apt install centreon-plugin-cloud-azure-datafactory-api
...
 
-Please find the troubleshooting documentation for the API-based plugins in
-this [chapter](../getting-started/how-to-guides/troubleshooting-plugins.md#http-and-api-checks).
\ No newline at end of file
+<Tabs groupId="sync">
+<TabItem value="Factory-Usage" label="Factory-Usage">
+
+| Option              | Description                                                                                                                          |
+|:--------------------|:-------------------------------------------------------------------------------------------------------------------------------------|
+| --resource          | Set resource name or id (Required).                                                                                                  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:391]:
```suggestion
| --resource          | Set resource name or ID (Required).                                                                                                  |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 10]
```diff
@@ -135,98 +152,267 @@ apt install centreon-plugin-cloud-azure-datafactory-api
...
+| --resource-group    | Set resource group (Required if resource's name is used).                                                                            |
+| --warning-$metric$  | Thresholds warning ($metric$ can be: 'factory-percentage-usage', 'resource-percentage-usage', 'factory-size', 'resource-count').     |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:393]:
```suggestion
| --warning-$metric$  | Warning threshold ($metric$ can be: 'factory-percentage-usage', 'resource-percentage-usage', 'factory-size', 'resource-count').     |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 11]
```diff
@@ -135,98 +152,267 @@ apt install centreon-plugin-cloud-azure-datafactory-api
...
+| --critical-$metric$ | Thresholds critical ($metric$ can be: 'factory-percentage-usage', 'resource-percentage-usage', 'factory-size', 'resource-count').    |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:394]:
```suggestion
| --critical-$metric$ | Critical threshold ($metric$ can be: 'factory-percentage-usage', 'resource-percentage-usage', 'factory-size', 'resource-count').    |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 12]
```diff
@@ -135,98 +152,267 @@ apt install centreon-plugin-cloud-azure-datafactory-api
...
+
+</TabItem>
+<TabItem value="Integration-Runtime" label="Integration-Runtime">
+
+| Option              | Description                                                                                                                                                                                                                                |
+|:--------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --resource          | Set resource name or id (Required).                                                                                                                                                                                                        |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:401]:
```suggestion
| --resource          | Set resource name or ID (Required).                                                                                                                                                                                                        |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 13]
```diff
@@ -135,98 +152,267 @@ apt install centreon-plugin-cloud-azure-datafactory-api
...
+| --resource-group    | Set resource group (Required if resource's name is used).                                                                                                                                                                                  |
+| --filter-metric     | Filter metrics (Can be: 'IntegrationRuntimeAvailableMemory', 'IntegrationRuntimeAvailableNodeNumber', 'IntegrationRuntimeAverageTaskPickupDelay', 'IntegrationRuntimeCpuPercentage', 'IntegrationRuntimeQueueLength') (Can be a regexp).   |
+| --warning-$metric$  | Thresholds warning ($metric$ can be: 'available-memory', 'available-node-number', 'average-task-pickup-delay', 'cpu-percentage', 'queue-length').                                                                                          |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:404]:
```suggestion
| --warning-$metric$  | Warning threshold ($metric$ can be: 'available-memory', 'available-node-number', 'average-task-pickup-delay', 'cpu-percentage', 'queue-length').                                                                                          |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 14]
```diff
@@ -135,98 +152,267 @@ apt install centreon-plugin-cloud-azure-datafactory-api
...
+| --critical-$metric$ | Thresholds critical ($metric$ can be: 'available-memory', 'available-node-number', 'average-task-pickup-delay', 'cpu-percentage', 'queue-length').                                                                                         |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:405]:
```suggestion
| --critical-$metric$ | Critical threshold ($metric$ can be: 'available-memory', 'available-node-number', 'average-task-pickup-delay', 'cpu-percentage', 'queue-length').                                                                                         |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 15]
```diff
@@ -134,98 +151,270 @@ yum install centreon-plugin-Cloud-Azure-DataFactory-Api
...
-Rendez-vous sur la [documentation dédiée](../getting-started/how-to-guides/troubleshooting-plugins.md#http-and-api-checks)
-des plugins basés sur HTTP/API.
+<Tabs groupId="sync">
+<TabItem value="Factory-Usage" label="Factory-Usage">
+
+| Option              | Description                                                                                                                          |
+|:--------------------|:-------------------------------------------------------------------------------------------------------------------------------------|
+| --resource          | Set resource name or id (Required).                                                                                                  |
+| --resource-group    | Set resource group (Required if resource's name is used).                                                                            |
+| --warning-$metric$  | Thresholds warning ($metric$ can be: 'factory-percentage-usage', 'resource-percentage-usage', 'factory-size', 'resource-count').     |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:395]:
```suggestion
| --warning-$metric$  | Warning threshold ($metric$ can be: 'factory-percentage-usage', 'resource-percentage-usage', 'factory-size', 'resource-count').     |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 16]
```diff
@@ -134,98 +151,270 @@ yum install centreon-plugin-Cloud-Azure-DataFactory-Api
...
+| --critical-$metric$ | Thresholds critical ($metric$ can be: 'factory-percentage-usage', 'resource-percentage-usage', 'factory-size', 'resource-count').    |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:396]:
```suggestion
| --critical-$metric$ | Critical threshold ($metric$ can be: 'factory-percentage-usage', 'resource-percentage-usage', 'factory-size', 'resource-count').    |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 17]
```diff
@@ -134,98 +151,270 @@ yum install centreon-plugin-Cloud-Azure-DataFactory-Api
...
+/usr/lib/centreon/plugins/centreon_azure_datafactory_factories_api.pl \
+	--plugin=cloud::azure::datafactory::factories::plugin \
+	--mode=integration-runtime \
+	--custommode='api' \
+	--resource='' \
+	--resource-group='' \
+	--subscription='' \
+	--tenant='' \
+	--client-id='' \
+	--client-secret='' \
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:259]:
```suggestion
	--resource='FACTORY001ABCD' \
	--resource-group='RSG1234' \
	--subscription='xxxxxxxxx' \
	--tenant='xxxxxxxxx' \
	--client-id='xxxxxxxxx' \
	--client-secret='xxxxxxxxx' \
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md (hunk 2 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:350]:
```suggestion
| --source-encoding                          | Define the character encoding of the response sent by the monitored resource. Default: 'UTF-8'.                                                                                                                                                                                     |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 18]
```diff
@@ -134,98 +151,270 @@ yum install centreon-plugin-Cloud-Azure-DataFactory-Api
...
+| --per-sec                                  | Display the statistics based on a per-second period.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+
+#### Options des modes
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:384]:
```suggestion

#### Microsoft Azure Rest API specifics

- To connect to the Azure Rest API, you must register an application (follow the [How-to guide](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal)).
- The application needs the 'Monitoring Reader' role (See https://docs.microsoft.com/en-us/azure/azure-monitor/platform/roles-permissions-security#monitoring-reader).
- This custom mode is using the 'OAuth 2.0 Client Credentials Grant Flow'.
- For further information, visit https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-oauth2-client-creds-grant-flow.

#### Options des modes
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 19]
```diff
@@ -135,98 +152,267 @@ apt install centreon-plugin-cloud-azure-datafactory-api
...
+/usr/lib/centreon/plugins/centreon_azure_datafactory_factories_api.pl \
+	--plugin=cloud::azure::datafactory::factories::plugin \
+	--mode=integration-runtime \
+	--custommode='api' \
+	--resource='' \
+	--resource-group='' \
+	--subscription='' \
+	--tenant='' \
+	--client-id='' \
+	--client-secret='' \
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:257]:
```suggestion
	--resource='FACTORY001ABCD' \
	--resource-group='RSG1234' \
	--subscription='xxxxxxxxx' \
	--tenant='xxxxxxxxx' \
	--client-id='xxxxxxxxx' \
	--client-secret='xxxxxxxxx' \
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md (hunk 7 above)
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:348]:
```suggestion
| --source-encoding                          | Define the character encoding of the response sent by the monitored resource. Default: 'UTF-8'.                                                                                                                                                                                    |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md [hunk 20]
```diff
@@ -135,98 +152,267 @@ apt install centreon-plugin-cloud-azure-datafactory-api
...
+| --per-sec                                  | Display the statistics based on a per-second period.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+
+#### Modes options
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/cloud-azure-datafactory-factories.md:382]:
```suggestion

#### Microsoft Azure Rest API specifics

- To connect to the Azure Rest API, you must register an application (follow the [How-to guide](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal)).
- The application needs the 'Monitoring Reader' role (See https://docs.microsoft.com/en-us/azure/azure-monitor/platform/roles-permissions-security#monitoring-reader).
- This custom mode is using the 'OAuth 2.0 Client Credentials Grant Flow'.
- For further information, visit https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-oauth2-client-creds-grant-flow.

#### Modes options
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**